plugins/jobs/web_client/templates/jobCheckBoxMenu.pug

```diff
@@ -2,10 +2,16 @@ a.g-job-checkbox-menu-link(data-toggle="dropdown", data-target=`#g-job-checkbox-
   | #{title}
   i.icon-filter
 .dropdown(id=`g-job-checkbox-menu-${title}`)
-  ul.dropdown-menu(role="menu")
-    each checked, name in values
-      li(role="presentation")
-        label
-          input.g-job-filter-checkbox(type="checkbox", data-toggle="tooltip",
-              id=name, title=name, checked=checked)
-          span #{name}
+  .dropdown-menu(role="menu")
+    .checkbox
+      label
+        input.g-job-filter-checkall(type="checkbox", data-toggle="tooltip", checked=checkAllChecked)
+        | Check all
+    hr
```

Comment:
Minor nit, but my understanding is that the `hr` tag is no longer supposed to be used for presentation, only semantics. If this is just meant to provide a visual divider between sections, my vote would be to use a `border` style rule on one of the surrounding elements or even a `div` with a fixed height and background color.